requirements.txt

```diff
@@ -3,5 +3,5 @@ numpy
 git+https://github.com/lsst/daf_butler@main#egg=lsst-daf-butler
 git+https://github.com/lsst/utils@main#egg=lsst-utils
 git+https://github.com/lsst/resources@main#egg=lsst-resources
-git+https://github.com/lsst/pipe_base@main#egg=lsst-pipe-base
+git+https://github.com/lsst/pipe_base@tickets/DM-41606#egg=lsst-pipe-base
```

Comment:
Friendly reminder to change it back when you merge everything.

Reply:
There is a GitHub action that prevents merging until this is backed out.